hack/verify-vendor.sh

```diff
@@ -21,29 +21,16 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-readonly branch=${1:-${KUBE_VERIFY_GIT_BRANCH:-master}}
```

Comment:
are we just dropping these since the job runs faster now?

Reply:
same as above